### PR TITLE
Styling of batch view

### PR DESCRIPTION
**Pull request by matharden**
## Stories covered
- [As a monies clerk I want to see a batch of unprocessed payments](https://www.pivotaltracker.com/story/show/93918368) - 93918368

## Additional
Includes creation of base styles:
- vertical flow
- checkboxes
- table
- buttons

### CODE REVIEW

File: mtp_cashbook/assets-src/stylesheets/components/_table.scss [hunk 1]
```diff
@@ -0,0 +1,58 @@
+$bg-color: #fff;
+$bg-color-alt: #ededed;
+$bg-color-selected: #e3f0ff;
+$border-color: #979797;
+
+// Fixes
+
+.numeric {
+  font-family: ntatabularnumbers;
```
Comment (SteveMarshall) [inline mtp_cashbook/assets-src/stylesheets/components/_table.scss]: :+1:

File: mtp_cashbook/templates/cashbook/transaction_batch_list.html [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+  {% if object_list %}
+  <div class="batch">
+    <table>
+      <thead>
+        <tr>
+          <th>Prisoner no</th>
+          <th>Prisoner name</th>
+          <th>Amount</th>
+          <th>Sender</th>
+          <th>Credited?</th>
```
Comment (SteveMarshall) [inline mtp_cashbook/templates/cashbook/transaction_batch_list.html]: Can we get the strings in here wrapped in `trans` and `blocktrans`?

Comment (matharden) [inline mtp_cashbook/templates/cashbook/transaction_batch_list.html]: Yes. Thanks, forgot about that.

File: mtp_cashbook/templates/cashbook/transaction_batch_list.html [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+        </tr>
+      </thead>
+
+      <tfoot>
+        <tr>
+          <th colspan="5" style="padding:0"></th>
+        </tr>
```
Comment (SteveMarshall) [inline mtp_cashbook/templates/cashbook/transaction_batch_list.html]: Do we need the inline style here?

**Comment by SteveMarshall** [comment]
@marcofucci Any thoughts on this?

**Comment by marcofucci** [comment]
looks good apart from the trans and blocktrans

File: mtp_cashbook/templates/cashbook/transaction_batch_list.html (hunk 3 above)
Comment (matharden) [inline mtp_cashbook/templates/cashbook/transaction_batch_list.html]: Ha! Hell no! Well spotted.

**Pull request merged**